### PR TITLE
Add ports 8880, 8881, 8882 to tcp-client

### PR DESCRIPTION
**Pull request by Werner86**
Added ports 8880, 8881, 8882 to tcp-client to be able to use ESP32-Serial-Bridge without adjusting any ports (https://github.com/AlphaLima/ESP32-Serial-Bridge)

Tested ESP32-Serial bridge with FLARM (Bluetooth and WIFI)

### CODE REVIEW

**Comment by MaxKellermann** [comment]
This PR branch needs cleanup. "Revert", "removed patch file", ... doesn't sound it's even worth looking at the commits.

**Comment by Werner86** [comment]
Hi Max,
what do you prefer, a new PR for this with the feature or what do you mean with cleanup?

**Comment by MaxKellermann** [comment]
Never a new PR, this loses all review history.
Just (force-) push the cleanups to your existing PR branch.

File: patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+@@ -270,6 +270,9 @@ FillTCPPorts(DataFieldEnum &dfe)
+   dfe.addEnumText(_T("4352"), 4352);
+   dfe.addEnumText(_T("2000"), 2000);
+   dfe.addEnumText(_T("23"), 23);
++  dfe.addEnumText(_T("8880"), 8880);
++  dfe.addEnumText(_T("8881"), 8881);
++  dfe.addEnumText(_T("8882"), 8882);
+ }
+ 
+ static void
```
Comment (kedder) [inline patch:14]: This file (`patch`) clearly does not belong to this PR

**Comment by Werner86** [comment]
> Never a new PR, this loses all review history.
> Just (force-) push the cleanups to your existing PR branch.

ok, done as requested :-)

File: patch (hunk 1 above)
Comment (Werner86) [reply patch]: file has been removed

**Comment by MaxKellermann** [comment]
v6.8.x: 4e901b56f68af5cd2da7e19d5bf1559b64933606
